ec/src/models/short_weierstrass_jacobian.rs

```diff
@@ -228,12 +216,6 @@ impl<P: Parameters> AffineCurve for GroupAffine<P> {
         })
     }
 
-    #[inline]
-    fn mul<S: Into<<Self::ScalarField as PrimeField>::BigInt>>(&self, by: S) -> GroupProjective<P> {
-        let bits = BitIteratorBE::new(by.into());
-        self.mul_bits(bits)
-    }
-
     #[inline]
     fn mul_by_cofactor_to_projective(&self) -> Self::Projective {
         self.scale_by_cofactor()
```

Comment:
Why do they have both `mul_by_cofactor_to_projective` and `scale_by_cofactor` if they do the exact same thing?

Reply:
Update: I removed `scale_by_cofactor`